samples/place-autocomplete-data-session/index.ts

```diff
@@ -5,175 +5,145 @@
 let newestRequestId = 0;
 let tokenCount = 0;
 
-async function initMap() {
-  await google.maps.importLibrary("maps");
-
-  innerMap = mapElement.innerMap;
-  innerMap.setOptions({
-    mapTypeControl: false,
-  });
-
-  // Create an initial request body.
-  request = {
-    input: "",
+// Create an initial request body.
+const request: google.maps.places.AutocompleteRequest = {
+    input: '',
     includedPrimaryTypes: [
-      "restaurant",
-      "cafe",
-      "museum",
-      "park",
-      "botanical_garden",
+        'restaurant',
+        'cafe',
+        'museum',
+        'park',
+        'botanical_garden',
     ],
-    language: "en-US",
-    region: "us",
-  };
-
-  // Update request center and bounds when the map finishes loading.
-  // google.maps.event.addListenerOnce(innerMap, 'idle', async () => {
-  //     // Update the request from the innerMap.
-  //     request!.locationRestriction = innerMap.getBounds();
-  //     request!.origin = innerMap.getCenter();
-
-  //     // Refresh the session token.
-  //     await refreshToken(request);
-  // });
-
-  // Update request center and bounds when the map bounds change.
-  google.maps.event.addListener(innerMap, "bounds_changed", async () => {
-    request!.locationRestriction = innerMap.getBounds();
-    request!.origin = innerMap.getCenter();
-  });
-
-  inputElement.addEventListener("input", makeAutocompleteRequest);
 }
 
-async function makeAutocompleteRequest(inputEvent) {
-  const { AutocompleteSuggestion } = (await google.maps.importLibrary(
-    "places"
-  )) as google.maps.PlacesLibrary;
-
-  // To avoid race conditions, store the request ID and compare after the request.
-  const requestId = ++newestRequestId;
-
-  // If the request is not initialized, do not proceed.
-  if (!request) return;
-
-  if (!request.sessionToken) {
-    await refreshToken(request);
-  }
-
-  // Reset elements and exit if an empty string is received.
-  if (
-    !inputEvent.target ||
-    (inputEvent.target as HTMLInputElement).value === ""
-  ) {
-    titleElement.innerText = "";
-    resultsContainerElement.replaceChildren();
-    return;
-  }
+async function initMap() {
+    await google.maps.importLibrary('maps');
+    innerMap = mapElement.innerMap;
+    innerMap.setOptions({
+        mapTypeControl: false,
+    });
 
-  // Add the latest char sequence to the request.
-  request.input = (inputEvent.target as HTMLInputElement).value;
+    // Update request center and bounds when the map bounds change.
+    google.maps.event.addListener(innerMap, 'bounds_changed', async () => {
+        request.locationRestriction = innerMap.getBounds();
+        request.origin = innerMap.getCenter();
+    });
 
-  // Fetch autocomplete suggestions and show them in a list.
-  const { suggestions } =
-    await AutocompleteSuggestion.fetchAutocompleteSuggestions(request);
+    inputElement.addEventListener('input', makeAutocompleteRequest);
```

Comment:
IIRC, we had a refreshToken call here? 

```suggestion
await refreshToken();

    inputElement.addEventListener('input', makeAutocompleteRequest);
```